plugins/DigitalLibrary/tests/TestDigitalFactoryFileModel.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+from src.DigitalFactoryFileModel import DigitalFactoryFileModel
+from src.DigitalFactoryFileResponse import DigitalFactoryFileResponse
+
+
+file_1 = DigitalFactoryFileResponse(client_id ="client_id_1",
+                                    content_type = "zomg",
+                                    file_name = "file_1.3mf",
+                                    file_id = "file_id_1",
+                                    library_project_id = "project_id_1",
+                                    status = "test",
+                                    user_id = "user_id_1",
+                                    username = "username_1",
+                                    uploaded_at = "2021-04-07T10:33:25.000Z")
+
+file_2 = DigitalFactoryFileResponse(client_id ="client_id_2",
+                                    content_type = "zomg",
+                                    file_name = "file_2.3mf",
+                                    file_id = "file_id_2",
+                                    library_project_id = "project_id_2",
+                                    status = "test",
+                                    user_id = "user_id_2",
+                                    username = "username_2",
+                                    uploaded_at = "2021-02-06T09:33:22.000Z")
+
+file_3 = DigitalFactoryFileResponse(client_id ="client_id_1",
```

Comment:
It's nice to give variables a descriptive name to indicate what makes their content relevant.

For file 1 and 2, as far as I could see, it's just that you needed 2 different files. For file 3, the point is that the file has the extension `wtf` instead of `3mf`. So perhaps this would be a better name for it:

```suggestion
file_wtf = DigitalFactoryFileResponse(client_id ="client_id_1",
```

(And change references to this name elsewhere too.)